QATCH/processors/CurveOptimizer.py

```
@@ -513,7 +528,13 @@ class DropEffectCorrection(CurveOptimizer):
 
     TAG = "[DropEffectCorrection]"
 
-    def __init__(self, file_path: str, file_buffer, initial_diff_factor: float = Constants.default_diff_factor, bounds: list = []):
+    def __init__(
+        self,
+        file_path: str,
+        file_buffer,
+        initial_diff_factor: float = Constants.default_diff_factor,
+        bounds: list = [],
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)